src/js/providers/html5.js

```diff
@@ -744,7 +744,11 @@ define([
 
         this.setContainer = function(containerElement) {
             _container = containerElement;
-            containerElement.appendChild(_videotag);
+            if (containerElement.firstChild) {
```

Comment:
Passing `null` to the first arg of `insertBefore` is allowed; this can be simplified to `containerElement.insertBefore(containerElement.firstChild, _videotTag)`